install_config/aggregate_logging.adoc

```diff
@@ -937,6 +962,65 @@ existing `journald` log driver configuration. It is recommended to use the
 Logging Driver] for instructions to change your existing log driver
 configuration to `json-file`.
 
+[[fluentd-file]]
+*Configuring Fluentd to Send Logs to a File*
+
+Fluentd writes logs to a file you specify with the `LOGGING_FILE_PATH` environment variable, by default `/var/log/fluentd/fluentd.log`.  
+When the current Fluentd log file reaches a specified size, {product-title} automatically renames the *fluentd.log* log file so that new logging data can be collected.
+You can control how many of the renamed files that {product-title} retains. 
+
+The following example shows logs in a cluster where the maximum log size is 1Mb and four logs should be retained. When the *fluentd.log* reaches 1Mb, {product-title} deletes the current *fluentd.log.4*, renames the each of the Fluentd in turn, and creates a new *fluentd.log*.  
+
+----
+fluentd.log     0b
+fluentd.log.1  1Mb
+fluentd.log.2  1Mb
+fluentd.log.3  1Mb
+fluentd.log.4  1Mb
+----
+
+Use the *logs* utility to print out the contents of Fluentd log files. For example:
+
+----
+oc exec <pod> logs <1>
+----
+
+<1> Specify the name of the Fluentd or Elasticsearch pod.
+
+For example:
+
+----
+oc exec logging-es-data-master-2-deploy logs
+oc exec logging-kibana-1 logs
+----
+
+The contents of log files are printed out, starting with the oldest log.  Use `-f` option to follow what is being written into the logs.
+
+To control the size of the Fluentd logs, you can specify a maximum log size and the number of logs that {product-title} retains. 
+ 
+* The `LOGGING_FILE_SIZE` parameter specifies how large the file can be before {product-title} renames the files. 
+
+* The `LOGGING_FILE_AGE` parameter specifies how many of the log files can be retained before {product-title} deletes the files. 
+
+.Parameters for configuring Fluentd logging
+[cols="3,7",options="header"]
+|===
+|Parameter
+|Description
+
+| `LOGGING_FILE_PATH` | The path to the Fluentd log file. If you want Fluentd to output to the console, `STDOUT`, set this variable to `console`. The default value is `/var/log/fluentd/fluentd.log`.
+| `LOGGING_FILE_SIZE` | The maximum size of a single Fluentd log file in Bytes. If the size of the *flientd.log* file exceeds this value, {product-title} renames the *fluentd.log.** files and creates a new *fluentd.log*. The default is 1024000 (1MB).
+| `LOGGING_FILE_AGE` | The number of logs that Fluentd retains before deleting. The default value is `10`. 
+|===
+
+For example:
+
+----
+LOGGING_FILE_PATH=console
+LOGGING_FILE_AGE=30
+LOGGING_FILE_SIZE=1024000
+----
```

Comment:
I think it's better to illustrate how to set these parameters in the doc.

Reply:
It doesn't make any sense to use `LOGGING_FILE_PATH=console` with `LOGGING_FILE_AGE` and `LOGGING_FILE_SIZE`

Reply:
I think it wold be better to simply exclude 'LOGGING_FILE_PATH' all together.  I would expect customer's to never really modify this location